Gemfile

```diff
@@ -22,6 +22,7 @@ gem 'jbuilder', '~> 2.0'
 gem 'sdoc', '~> 0.4.0', group: :doc
 
 gem 'devise'
+gem 'activeadmin', github: 'gregbell/active_admin'
```

Comment:
i've enjoyed using `activeadmin` - it's pretty easy to use, and i hope you've found it to be, as well